src/pages/dar_application/RequiredDAAs.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export default function RequiredDAAs(props) {
+  const { datasets, daas, daaDownload } = props;
+  const fileNames = new Set();
+  const daaDivs = datasets.map((dataset) => {
+    const datasetDacId = dataset.dacId;
+    if (!datasetDacId) {
+      return <div key={dataset.id}></div>;
+    }
+    const daa = daas.find((daa) => daa.dacs.some((d) => d.dacId === datasetDacId));
+    const id = daa.daaId;
+    const fileName = daa.file.fileName.split('.')[0];
+    if (fileNames.has(fileName)) {
+      return <div key={id-dataset.name}></div>;
+    }
+    fileNames.add(fileName);
+    return (
+      <div key={id}>
+        {daaDownload(id, fileName)}
+      </div>
+    );
+  });
+  if (fileNames.size === 0) {
+    return (
+      <div></div>
+    );
```

Comment:
If we release this to prod, all DARs will no longer show any data access agreements. I want to confirm that this is intended behavior. The SO DAA screens are currently flagged on the environment, but this is not so we are in an interim state until the SO DAA screens are in production.

Reply:
That's a great point, I don't think we would want no DAAs showing up if we release this to prod but I'm not positive. I propose that we wait until the SO DAA screens are in production before we release this to prod to ensure all parts necessary for this change to make sense are released together.

Reply:
My concern here has been resolved with the conditional render. This will only be displayed when we fully release this to higher environments.